docs/src/main/docs/guides/07_datasource.adoc

```diff
@@ -0,0 +1,364 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+= Setting Up Data Sources
+:description: Helidon MP Data Source Guide
+:keywords: helidon, guide, datasource, microprofile
+
+This guide shows how to configure and use named
+https://docs.oracle.com/javase/8/docs/api/javax/sql/DataSource.html[`DataSource`]s
+in your Helidon MP application.
+
+== What You Need
+
+|===
+|About 20 minutes
+|<<about/03_prerequisites.adoc,Helidon Prerequisites>>
+|The Maven coordinates for your database vendor's JDBC-compliant database driver library
+|===
+
+== Add Your Database Driver to Your Helidon MP Application's Runtime Classpath
+
+Make sure your database vendor's driver library is present on your
+application's runtime classpath.  In a Maven project, add an
+appropriate
+https://maven.apache.org/ref/3.6.1/maven-model/maven.html#class_dependency[`<dependency>`]
```

Comment:
Referencing the maven model documentation for `dependency` seems a bit overkill.
Just saying "add a dependency" is enough.